pandas/tests/io/formats/test_format.py

```diff
@@ -1636,9 +1636,9 @@ def test_to_string_complex_float_formatting(self):
             df = DataFrame(
                 {
                     "x": [
-                        (0.4467846931321966 + 0.0715185102060818j),
-                        (0.2739442392974528 + 0.23515228785438969j),
-                        (0.26974928742135185 + 0.3250604054898979j),
+                        (0.4_467_846_931_321_966 + 0.0_715_185_102_060_818j),
+                        (0.2_739_442_392_974_528 + 0.23_515_228_785_438_969j),
```

Comment:
This is not how I do numbers after the dot. I'd do ``0.446_784_693_132_196_6`` etc.